### DISCUSSION TITLE
Being a test user

### DISCUSSION THREAD

**Discussion by szepeviktor** [szepeviktor/wordpress-website-lifecycle#2] [Ideas]
Hello @montchr! 👋🏻 
Thank you for the star.

Would you be a test user for this repo?
I'd like to measure how easily WordPress people understand modernized work with WordPress.

**Comment by szepeviktor** [comment]
@mircobabini Hello!
Would you help me with some feedback? I'm bad at explaining things.

**Reply by mircobabini** [reply]
Hey @szepeviktor! Thanks for opening this discussion.

I think you already did a great job writing this docs. I think it can be enhanced in these areas:
- When referring to CI/CD, provide an example of a plugin which can be deployed with CI/CD automatically (reference to a ready to use github action for ssh/ftp deploy)
- Say that `child-theme-configurator` can be disabled after child theme creation
- WP-Cron by cron-job: provide an example (note that `wp cron event run --due-now` is not working on some envs, fallback to the `curl` version)
- PSR-4 autoloading example
- Child Theme: Divi is a paid theme. I would suggest to use a free theme for this docs (like [hello child](https://github.com/elementor/hello-theme-child))
- tests example or reference to https://github.com/lucatume/wp-browser / https://codeception.com/
- a reference to the serverless options out there (like https://ymirapp.com/)

**Reply by mircobabini** [reply]
I currently have set a study sessions in January. Gonna test it out deeply.

**Reply by szepeviktor** [reply]
@mircobabini Thank you!
Added docs, GHA workflow and deploy script: https://github.com/szepeviktor/composer-managed-wordpress/blob/master/INSTALL.md#continuous-delivery-cd

**Reply by szepeviktor** [reply]
- disabling a plugin is kindergarten level stuff, so that is must-have knowledge
- WP-Cron also went into INSTALL.md
- PSR-4 example is in https://github.com/szepeviktor/small-project/blob/035e4ede5bbe7cf871ac4e3165eedc06df125ba3/composer.json#L27-L34 (example plugin repo)
- the child theme (and everything else) is not for copy-pasting
- wp-brower needs to be rewritten to qualify as "a software"
- serverless prevents you from taking control, I run web applications every day

**Reply by mircobabini** [reply]
Ok, please note that Ymir is just a cli tool to deploy a site on AWS; the deployed website is then using AWS Lambda to leverage the scalability, then it's serverless by definition. But you still have 100% control on your application, I'd say.

**Reply by szepeviktor** [reply]
> still have 100% control

AFAIK AWS Lambda can be configured but you are not root/sysadmin.
My way is to trust UpCloud with the infrastructure and build everything on an empty Debian OS.
https://github.com/szepeviktor/wordpress-website-lifecycle/blob/master/WordPress-stack.md 👀

**Reply by mircobabini** [reply]
Yeah ok, now I got what you mean. Yeah, I see the whole codebase refers to an env where you have full root control.

**Reply by szepeviktor** [reply]
Thank you for your contribution!! 😍 
It is very valuable.

Yes, I work as an "error prevention officer" building nothing. All I do is talking about future errors. That needs _full control_ from the OS up to the user's browser.

**Comment by szepeviktor** [comment]
Hello @brianvarskonst! 👋🏻 

Is it possible to understand my writings? Could you help improving this repository?

**Comment by szepeviktor** [comment]
👋🏻 Hello @djrasmusp!
I see you're working with WordPress. Could you help me make this repo easier to understand?

**Reply by djrasmusp** [reply]
Hey @szepeviktor - I will get it a try, just have to find time to look though the repo and your descriptions. But would love to help you and this repo
